Wikipedia/Code/MWKSiteInfo.m

```diff
@@ -14,16 +14,21 @@ typedef NS_ENUM(NSUInteger, MWKSiteInfoNSCodingSchemaVersion) {
 @interface MWKSiteInfo ()
 @property (readwrite, copy, nonatomic) NSURL *siteURL;
 @property (readwrite, copy, nonatomic) NSString *mainPageTitleText;
+
```

Comment:
Here's where the `readwrite` versions of `readingListsConfigMaxEntriesPerList` and `readingListsConfigMaxListsPerUser` can go. That way internally we can write to them but externally they're immutable.